pangeo_forge_runner/commands/bake.py

```diff
@@ -171,6 +172,21 @@ def autogenerate_job_name(self):
 
         return job_name
 
+    def add_unique_suffix_to_flink_jobs(self, per_recipe_unique_job_name):
+        """FlinkOperatorBakery job names always need to be unique
+
+        to accommodate reruns and race conditions (two users running same recipe)
+        """
+        if self.bakery_class == FlinkOperatorBakery:
+            unique_suffix = "".join(
+                secrets.choice(string.ascii_letters + string.digits) for _ in range(5)
```

Comment:
Not sure that it matters, but I like random for this kind of thing so that a seed can be used for predictable outputs